CMakeLists.txt

```diff
@@ -762,3 +762,8 @@ endif()
 
 install(DIRECTORY doc/man1 DESTINATION ${CMAKE_INSTALL_MANDIR}
   FILES_MATCHING PATTERN "*.1")
+
+if(opm-common_EMBEDDED_PYTHON)
+    set_target_properties(flow PROPERTIES
+                        ENVIRONMENT "PYTHONPATH=${PROJECT_BINARY_DIR}/python:${opm-common_DIR}/python:$ENV{PYTHONPATH}")
```

Comment:
this makes little sense. you cannot set the environment for an executable, only on a test.

Reply:
ok, thanks! That explains why that has no effect.. I also tried setting the environment variable in the shell script that execute the failing tests: https://github.com/OPM/opm-simulators/pull/5306/commits/fd2daa286f720c3d02bf2db8569ed0618e4225a9, but that also did not work

Reply:
yeah, i just checked it and that code does the intended job at least.